docs/user_manual/source/rvfi.rst

```diff
@@ -214,6 +214,38 @@ The ``pc_wdata`` signal shows the predicted next program counter. This predictio
 
 **Memory Access**
 
+For |corev|, the rvfi_mem interface has been expanded to support multiple memory operations per instruction. The new format of the rvfi_mem signals can be seen in the table below.
+
+.. code-block:: verilog
+
+   output [NRET * NMEM * XLEN - 1 : 0]   rvfi_mem_addr
+   output [NRET * NMEM * XLEN/8 - 1 : 0] rvfi_mem_rmask
+   output [NRET * NMEM * XLEN/8 - 1 : 0] rvfi_mem_wmask
+   output [NRET * NMEM * XLEN - 1 : 0]   rvfi_mem_rdata
+   output [NRET * NMEM * XLEN - 1 : 0]   rvfi_mem_wdata
+
+Any instruction that issues multiple loads or stores will populate multiple entries in rvfi_mem. Misaligned loads or stores that are split by |corev| due to alignment will be reported once, in the lowest index of rvfi_mem.
+Instructions will populate the rvfi_mem outputs with incrementing ``NMEM``, starting at ``NMEM=1``. Instructions with a single memory operation, including split misaligned transfers, will pouplate rvfi_mem as follows:
```

Comment:
I think all text from 'Instructions with a single memory operation' and onwards can be removed especially because people won't know details about a 'memory operation counter'

Reply:
Do you mean everything from "instructions with a single memory operation" all the way down to "**CSR Signals** ?

Reply:
Yes, isn't all that just giving two examples of value of NMEM (the MMEM = 1 case and the MMEM > 1 case)?

Reply:
Maybe you can say something as follows instead:

Instructions with a single memory operation (e.g. all RV32I instructions), including split misaligned transfers, will only use NMEM = 1. Instructions with multiple memory operations (e.g. the push and pop instructions from Zcmp) use NMEM > 1 in case multiple memory operations actually occur.

Reply:
I like your last suggestion, changed to that.